### PR TITLE
Iss1281

### PR DESCRIPTION
**Pull request by hobbit1983**

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.ipnetwork.manager/src/main/java/dev/galasa/ipnetwork/spi/SSHClient.java [hunk 1]
```diff
@@ -254,7 +250,6 @@ public String issueCommandToShell(String command, boolean newShell) throws SSHEx
      *                 before timing out
      * @return the output of the command
      * @throws SSHException
-     * @throws JSchException
      */
     @Override
     public synchronized String issueCommandToShell(String command, boolean newShell, long timeout) throws SSHException {
```
Comment (techcobweb) [inline galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.ipnetwork.manager/src/main/java/dev/galasa/ipnetwork/spi/SSHClient.java:255]: Might be more helpful to call the last parameter timeoutSeconds ? Then the user knows the units of the timeout also ?

File: galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.ipnetwork.manager/src/main/java/dev/galasa/ipnetwork/spi/SSHClient.java [hunk 2]
```diff
@@ -613,7 +605,7 @@ public void run() {
 
                     long timeout = System.currentTimeMillis() - idleTimeout;
                     if (timeout >= lastCommandTimestamp) {
-                        logger.debug("No command issued after " + idleTimeout + " milliseconds, closing SSH session");
+                        logger.debug("SSH Client unused after " + idleTimeout + " milliseconds, freeing session");
```
Comment (techcobweb) [inline galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.ipnetwork.manager/src/main/java/dev/galasa/ipnetwork/spi/SSHClient.java:608]: Similarly with these timeout/idleTimeout variables. They look like MilliSecs to me...

**Review by techcobweb — APPROVED** [review]
I've approved, but suggested a few possible improvements...

**Comment by hobbit1983** [comment]
Thanks I will make those amendments!

File: galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.ipnetwork.manager/src/main/java/dev/galasa/ipnetwork/spi/SSHClient.java (hunk 1 above)
Comment (hobbit1983) [reply galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.ipnetwork.manager/src/main/java/dev/galasa/ipnetwork/spi/SSHClient.java:255]: good idea!

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**